api/tests/test_cloud_record_summary_get.py

```diff
@@ -157,20 +119,49 @@ def test_cloud_record_summary_get_200(self):
                              '"Day":31,'
                              '"Month":7}]}')
 
-        try:
-            # Check the expected response code has been received.
-            self.assertEqual(response.status_code, 200)
-            # Check the response received is as expected.
-            self.assertEqual(response.content, expected_response)
-            # Clean up after test.
-        finally:
-            self._clear_database(database)
-            database.close()
+        with self.settings(ALLOWED_FOR_GET='TestService',
+                           RETURN_HEADERS=["WallDuration",
+                                           "Day",
+                                           "Month",
+                                           "Year"]):
+            try:
+                self._check_summary_get(200,
+                                        expected_response=expected_response,
+                                        options=("?group=TestGroup"
+                                                 "&from=20000101&to=20191231"),
+                                        authZ_header_cont="Bearer TestToken")
+            finally:
+                # Clean up after test.
+                self._clear_database(database)
+                database.close()
 
     def tearDown(self):
         """Delete any messages under QPATH and re-enable logging.INFO."""
         logging.disable(logging.NOTSET)
 
+    def _check_summary_get(self, expected_status, expected_response=None,
+                           options='', authZ_header_cont=None):
+        """Helper method to make a GET request."""
+        test_client = Client()
+        # Form the URL to make the GET request to
+        url = ''.join((reverse('CloudRecordSummaryView'), options))
+
+        # If content for a HTTP_AUTHORIZATION has been provided,
```

Comment:
I'd advise bumping this and the next comment down past their `if` or `else` and indenting them so that they're at the same level as the body of the branch they apply to.